trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class TrimmerDecorator < BaseDecorator
```

Comment:
- [ ] For this inheritance to work you need to import base_decorator into this file.

  ```
  require_relative './base_decorator'
  ```